docs/source/index.rst

```diff
@@ -3,18 +3,186 @@
    You can adapt this file completely to your liking, but it should at least
    contain the root `toctree` directive.
 
-Welcome to ceph-medic's documentation!
-======================================
+=================================================
+ceph-medic -- Find common issues in Ceph clusters
+=================================================
 
-.. toctree::
-   :maxdepth: 2
-   :caption: Contents:
 
+``ceph-medic`` is a very simple tool to run against a Ceph cluster to detect
+common issues that might prevent correct functionality. It requires
+non-interactive SSH access to accounts that can ``sudo`` without a password
+prompt.
 
+Installation
+============
 
-Indices and tables
-==================
+``ceph-medic`` supports a couple different installation methods:
 
-* :ref:`genindex`
-* :ref:`modindex`
-* :ref:`search`
+github
```

Comment:
installing from github should be last I think, most will try to install with something else, like shaman (for now). And when we have actual releases we can get them onto ceph repos and that method should come first